tests/unit/test_api_patch.py

```diff
@@ -249,6 +215,10 @@ def test_patch_cannot_set_timestamps(client, maximal_contact):
     assert expected["products"] == []
     assert "products" not in actual
     actual["products"] = []
+    # The response shows computed fields for retro-compat. Contact schema
+    # does not have them.
+    del actual["vpn_waitlist"]
+    del actual["relay_waitlist"]
```

Comment:
Should we add some sort of TODO here to remove these values from the maximal contact when we're fully migrated?